README.md

```diff
@@ -1,12 +1,10 @@
-# Introduction
-NNFusion is a flexible and efficient DNN compiler that can generate high-performance executables from a DNN model description. NNFusion is designed to facilitate DNN compiler research, with full stack of optimizations built-in and can target different accelerator devices.
+**NNFusion** (Nerual Network Fusion) is a flexible and efficient DNN compiler that can generate high-performance executables from a DNN model description. NNFusion is designed to facilitate DNN compiler research, with full stack of optimizations built-in and can target different accelerator devices.
 
-### Major goal
-- Framework independent: support ONNX, TensorFlow and PyTorch models through a source-to-source (model to code) compilation
-- Innovation agility: provide a flexible and modular architecture to enable new compiler optimization research
-- Hardware neutral: aims to be able to support existing and future 1st and 3rd party accelerator devices
+## Who should consider using NNFusion?
+- Developers who want to generate high-performance executables from DNN model description of different frameworks including ONNX, TensorFlow and PyTorch.
+- Researchers who want to optimize DNN models and apply the method to their own reasearch work.
 
-### Highlight features
+## Highlight features
```

Comment:
please review this list with team, figure out:
- whether this list is the up to date full list
- are all of them been supported today